Changelog.md

```diff
@@ -20,8 +24,8 @@
     This is useful when reacting to noop errors.
 
 * [#1154](https://github.com/mbj/mutant/pulls/1154)
- 
-  * Add subcommand `environment subject list`. It allows to list 
+
+  * Add subcommand `environment subject list`. It allows to list
```

Comment:
My editor trimmed some trailing whitespace. I can remove this from the commit if you want but also seems like it is worth fixing.

Reply:
Added an extra commit, and my editors whitespace trimming broke ;) Good catch.